docs/setup-cluster/deploy-cluster/slurm/slurm-known-issues.rst

```diff
@@ -369,6 +369,20 @@ sometimes resolved by additionally installing the ``apptainer-setuid`` package.
    -  ``Resources``: Expected when resources are in use by other jobs. Otherwise, verify you have
       not requested more resources (GPUs, CPUs, nodes, memory) than are available in your cluster.
 
+.. _pbs-known-issues:
+
+******************
+ PBS Known Issues
```

Comment:
Amazing that we didn't have this section before :-)

In other sections we mostly describe the symptom first instead of the cause as the users are looking here for a reason presumably.   But we have multiple issues here with the same cause.   Maybe a first level sentence and then show the other cases as descriptions.

- Jobs are treated as successful even in the presences of a failure when PBS job history is not enabled.
   Without job history enabled, the launcher is unable to obtain the exit status of jobs and therefore they are all reported as successful.    This will prevent failed jobs from automatically restarting, and in the case of a job that fails to start running at all, it may be reported as completed with no error message reported.   Refer to [PBS Requirements](https://hpe-mlde.determined.ai/latest/setup-cluster/deploy-cluster/slurm/slurm-requirements.html#pbs-config-requirements)